rust/stackable-cockpit/src/platform/manifests.rs

```diff
@@ -74,6 +76,11 @@ pub trait InstallManifestsExt {
     ) -> Result<(), Error> {
         debug!("Installing manifests");
 
+        Span::current().pb_set_style(
+            &ProgressStyle::with_template("Progress: {wide_bar} {pos}/{len}").unwrap(),
+        );
+        Span::current().pb_set_length(manifests.len() as u64);
+
```

Comment:
These can be chained if you want:


```suggestion
        Span::current().pb_set_style(
            &ProgressStyle::with_template("Progress: {wide_bar} {pos}/{len}").expect("valid progress template"),
        )
        .pb_set_length(manifests.len() as u64);

```

Reply:
Unfortunately, those can't be chained, the `pb_*` functions don't have a return value

Reply:
No problem.